src/main/java/com/onaio/steps/Properties.java

```diff
@@ -31,16 +26,21 @@
 import static com.onaio.steps.helper.Constants.PA_FORM_ID;
 import static com.onaio.steps.helper.Constants.PA_MAX_AGE;
 import static com.onaio.steps.helper.Constants.PA_MIN_AGE;
+import static com.onaio.steps.helper.Constants.SETTINGS_AUTH_TIME;
+import static com.onaio.steps.helper.Constants.SETTINGS_PASSWORD_HASH;
+
+import java.util.HashMap;
+import java.util.Map;
 
 public class Properties {
-    private static final Map<String, String> props = new HashMap<String, String>();
+    private static final Map<String, String> props = new HashMap<>();
 
     static {
-        props.put(ENDPOINT_URL, "");
+        props.put(ENDPOINT_URL, "https://preview.steps.ona.io/upload-file");
```

Comment:
Is the the final production URL?

Reply:
I don't think so but @ciremusyoka can confirm. I will also move the pre-load URL in `build.gradle` file.

Reply:
@owais-vd this is not the production WHO Qatar URL. This is https://swa.moph.gov.qa

Reply:
Cool, I'll update the URL for debug and release build.

**For Debug:** https://preview.steps.ona.io

**For Release:** https://swa.moph.gov.qa

Reply:
No maintain the current URLs as they are on the Play Store version. Other will enter the correct URL on their apps

Reply:
i've maintained the current release URL based on this https://github.com/onaio/steps-app/pull/160#discussion_r841653629

Reply:
@ciremusyoka needs your input on https://github.com/onaio/steps-app/pull/160#discussion_r848491252

Reply:
@bkimondiu needs your input on https://github.com/onaio/steps-app/pull/160#discussion_r848491252

Reply:
@owais-vd I agree with @ekigamba that we shouldn't use MOPH Qatar's URLs as the defaults on the app. Could we retain the URLs as they are and later change them to point to the instance that WHO Global will deploy? cc: @ukanga

Reply:
ok, currently we are using https://preview.steps.ona.io.